src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -21,6 +23,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 
+import cucumber.api.java.Before;
```

Comment:
Do you need to import this? (cucumber.api.java.Before)

Reply:
Removed